samples/net/ptp/slave/src/ptp_slave.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2021 Carbon Robotics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <net/net_if.h>
+#include <net/ptp_time.h>
+
+#define PTP_MESSAGE_TYPE_SYNC 0x0
```

Comment:
Various defines in this file are already found in `include/net/gptp.h` file, and if something is missing there then those can be added to that header file. I would rather see that the symbols in gptp.h are used here in the sample so we avoid defining same ones just for this sample.